### PR TITLE
Add terms of use to security alerts settings

### PR DESCRIPTION
**Pull request by segun**
## **Description**
This a request that came from legal after their review of the security alert feature. Within the security alerts settings, we need to:
1. add the following sentence at the end of the main description of the feature: `By enabling this feature you agree to the provider's terms of use.`
2. add `Terms of use` at the end of Blockaid description and make it a link for http://blockiad.io/legal/metamask-ppom-privacy-policy
3. add `Terms of use` at the end of OpenSea+Blockaid description and make it a link for https://opensea.io/securityproviderterms

## **Manual testing steps**

1. Checkout develop
2. Start MetaMask
3. See that the security alert description doesn't include the words in (1) above and there are no terms of use links
4. Checkout this branch
5. Reload MetaMask
6. See that the security alert description include the words in (1) above and there are links for blockaid and opensea terms of use.

### **Before**

Screenshot 2023-10-19 at 17 39 09

### **After**

Screenshot 2023-10-19 at 17 37 46

## **Related issues**

Fixes: [#1529](https://github.com/MetaMask/MetaMask-planning/issues/1529)

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained:
  - [x] What problem this PR is solving.
  - [x] How this problem was solved.
  - [x] How reviewers can test my changes.
- [x] I’ve indicated what issue this PR is linked to: Fixes #???
- [x] I’ve included tests if applicable.
- [x] I’ve documented any added code.
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)).
- [x] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by jpuri — DISMISSED** [review]

**Review by blackdevelopa — DISMISSED** [review]

**Review by blackdevelopa — DISMISSED** [review]

**Review by georgewrmarshall — COMMENTED** [review]
Could we use ButtonLink for these links? <ButtonLink href="" externalLink size={ButtonLinkSize.Inherit}>

File: shared/lib/ui-utils.js [hunk 1]
```diff
@@ -11,3 +11,9 @@ export const ETHERSCAN_PRIVACY_LINK = 'https://etherscan.io/privacyPolicy';
 export const CONSENSYS_PRIVACY_LINK = 'https://consensys.net/privacy-policy/';
 export const AUTO_DETECT_TOKEN_LEARN_MORE_LINK =
   'https://consensys.net/privacy-policy/';
+
+///: BEGIN:ONLY_INCLUDE_IN(blockaid)
+export const BLOCKAID_TERMS_OF_USE =
+  'http://blockiad.io/legal/metamask-ppom-privacy-policy/';
```
Comment (seaona) [inline shared/lib/ui-utils.js:17]: There is a typo in `blockiad` so the URL is failing (see 1st pic).
However, if I fix it to `blockaid` I also see an error page (see 2nd pic). We should double-check this link

Screenshot from 2023-10-25 10-07-24

Screenshot from 2023-10-25 10-06-23

Comment (segun) [reply shared/lib/ui-utils.js:17]: I'll fix the typo, but @bschorchit mentioned the blockaid team is supposed to fix the page not found issue

Comment (seaona) [reply shared/lib/ui-utils.js]: thank you @segun , about the `http`, could we `https` instead? (if @bschorchit / Blockaid agrees)

**Review by jpuri — APPROVED** [review]

**Review by blackdevelopa — APPROVED** [review]

File: shared/lib/ui-utils.js (hunk 1 above)
Comment (segun) [reply shared/lib/ui-utils.js]: Waiting for blockaid team to fix the link before merging this.

**Pull request merged**